test/model/bpe_test.exs

```diff
@@ -0,0 +1,52 @@
+defmodule Tokenizers.Model.BPETest do
+  use ExUnit.Case, async: true
+  doctest Tokenizers.Model.BPE
+
+  describe "Initialized from memory" do
```

Comment:
Nitpick: make sure all `describe` and `test` start in lowercase to be consistent with the rest of the project. :)